packages/mcp/src/index.ts

```diff
@@ -197,6 +202,15 @@ You can also use special values to control access to orgs:
 
     const transport = new StdioServerTransport();
     await server.connect(transport);
+    
+    // Send SERVER_STARTED telemetry event
+    this.telemetry?.sendEvent('SERVER_STARTED', {
+      version: this.config.version,
+      toolsets: (flags.toolsets ?? []).join(', '),
+      orgs: sanitizeOrgInput(flags.orgs),
+      transport: 'stdio',
+    });
```

Comment:
we already have an event for successful server startups:
https://github.com/search?q=repo%3Asalesforcecli%2Fmcp%20%2FSERVER_START_SUCCESS%2F%20lang%3Ats&type=code

> orgs: sanitizeOrgInput(flags.orgs),

I'm pretty sure this is also captured for all events:

set here in the constructor:
https://github.com/salesforcecli/mcp/blob/373387504db9b1e6e583ed528dc47384b5a288d4/packages/mcp/src/index.ts#L151

set here (this method is called when the server is initialized:
https://github.com/salesforcecli/mcp/blob/373387504db9b1e6e583ed528dc47384b5a288d4/packages/mcp/src/telemetry.ts#L89